src/patterns/definitions.owl

```diff
@@ -7,8 +7,142 @@ Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)
+EquivalentClasses(<http://purl.obolibrary.org/obo/CL_4033005> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/CL_0000313> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_8410043>)))
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033006> (endothelial cell of lymphatic vessel of efferent lymphatic vessel)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:32094869"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033006> "A(n) endothelial cell of lymphatic vessel that is part of a(n) efferent lymphatic vessel."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> <http://purl.obolibrary.org/obo/CL_4033006> <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/CL_4033006> "endothelial cell of lymphatic vessel of efferent lymphatic vessel"^^xsd:string)
+EquivalentClasses(<http://purl.obolibrary.org/obo/CL_4033006> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/CL_0002138> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_0010397>)))
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033007> (brush cell of bronchus of epithelium of lobar bronchus)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:15817800"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033007> "A(n) brush cell of bronchus that is part of a(n) epithelium of lobar bronchus."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> <http://purl.obolibrary.org/obo/CL_4033007> <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/CL_4033007> "brush cell of bronchus of epithelium of lobar bronchus"^^xsd:string)
+EquivalentClasses(<http://purl.obolibrary.org/obo/CL_4033007> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/CL_0002208> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_0002339>)))
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033008> (vein endothelial cell of respiratory system venous endothelium)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:34030460"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033008> "A(n) vein endothelial cell that is part of a(n) respiratory system venous endothelium."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> <http://purl.obolibrary.org/obo/CL_4033008> <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/CL_4033008> "vein endothelial cell of respiratory system venous endothelium"^^xsd:string)
```

Comment:
Consider simplifying label as 'venous endothelial cell of respiratory system', which is easier to conceptualise and aligns with language used in the reference: "venous ECs".

Reply:
Having looked at this example, we agree that over-riding the automatically generated name with your suggestion would be best.  While analysing the ontology, we notice that the logical definition of blood vessel endothelial cell should be simplified from "[squamous epithelial cell](https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0000076) and [part of](https://www.ebi.ac.uk/ols/ontologies/cl/properties?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FBFO_0000050) some [blood vessel endothelium](https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0004638)" to just use 'endothelial cell' as a genus.